script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -1098,12 +1235,13 @@ I can't rely on someone as weak as you to do it alone, so that's why I offered t
 [sha:3bb589c11e464a000443e7900e72d380cfbebd8f]{
 -- Page 422, Offset 17627.
 -- アルクェイドは、ぱあ、と目を輝かした。
-Arcueid's eyes light up, sparkling.
+Arcueid's eyes light up.
 }
 [sha:fe96e84373da3090004dcab5761e499b34a35619]{
 -- Page 422, Offset 17628.
 -- そのまま、嬉しそうにこっちの手を握って、ぶんぶんと上下に振り回す。
-She grabs my hands and shakes them up and down with glee.
+// Doesn't say 両手 here so... assuming just the one hand for all these?
```

Comment:
that's not necessarily a great assumption given how damn flaky people are with this, but I think in this case you're right.